channels/unsorted.m3u

```diff
@@ -69,3 +69,7 @@ http://51.210.199.9/hls/stream.m3u8
 https://59d39900ebfb8.streamlock.net/streetmusic/streetmusic/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",نوبل TV
```

Comment:
That's Noble TV from Egypt
![image](https://user-images.githubusercontent.com/5430116/121944845-cd98e700-cd5b-11eb-9fda-d0524faa6f39.png)